.secrets.baseline

```diff
@@ -1,9 +1,4 @@
 {
```

Comment:
The changes in this file should have been in another PR. If you are changing something unrelated to the task, make another PR for it, so that it's easier to roll back.

But since this is a change that only affects development, it's ok.

Reply:
Oh actually I tried to but I can't commit my changes without adding this file
(I think it might be due to the precommit hook for `detect-secrets`?)

Reply:
Could be because of the version mismatch between your local version (1.1.0) of `detect-secrets` and the version that `.secrets.baseline` was generated with (0.14.3).